src/Microsoft.AspNetCore.Http/HttpContextFactory.cs

```diff
@@ -43,18 +43,12 @@ public HttpContext Create(IFeatureCollection featureCollection)
                 throw new ArgumentNullException(nameof(featureCollection));
             }
 
-            var responseCookiesFeature = new ResponseCookiesFeature(featureCollection, _builderPool);
-            featureCollection.Set<IResponseCookiesFeature>(responseCookiesFeature);
```

Comment:
Might be missing it with that example? It will create its own `ResponseCookiesFeature` with the `??`?